### PR TITLE
Update to JavaScriptKit 0.11.1

### PR DESCRIPTION
**Pull request by revolter**
I'm not sure that I did it correctly, but I need this change in order to use the latest version of Tokamak.

### CODE REVIEW

File: Sources/DOMKitDemo/main.swift [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import JavaScriptKit
 import DOMKit
 
-let document = global.document
+let document = globalThis.document
```
Comment (revolter) [inline Sources/DOMKitDemo/main.swift:4]: I'm especially not sure about this 😟

**Comment by MaxDesiatov** [comment]
Thanks for the PR, and sorry for the delay. It looks like SwiftWasm 5.5 release is imminent, and we'd like to update JavaScriptKit with support for `async`/`await`, which may make a lot of things in this PR outdated. It could be easier to update to that new future version if JSKit at once, skipping intermediate version. Let me know if that causes any inconvenience for you and if you'd like intermediate version such as 0.10 still be supported in some commit of DOMKit anyway

**Comment by revolter** [comment]
I didn't have time to continue working on my project lately 😞 But even so, I used my fork's branch in SPM, so you can safely close this PR 👍🏻

**Comment by MaxDesiatov** [comment]
Thank you for the PR, but we'll have to close it as outdated. DOMKit depends on the `main` branch of JSKit right, and we'll switch that dependency to a stable version as soon as that's released.

**Pull request closed without merging**